test/sumo_test_people.erl

```diff
@@ -56,7 +71,7 @@ sumo_sleep(Person) ->
     description => Person#person.description,
     profile_image => Person#person.profile_image}.
 
--spec sumo_wakeup(sumo:doc()) -> person().
+-spec sumo_wakeup(Person::sumo:doc()) -> sumo:user_doc().
```

Comment:
This is not correct, the result of `sumo_wakeup` (like the input of `sumo_sleep`) should be `person()`.